src/test/ensemble_config_change_stress.cpp

```diff
@@ -1011,6 +1025,9 @@ class StressPipelineConfigChanges : public TestWithTempDir {
     }
     void SetUp() override {
         TestWithTempDir::SetUp();
+        char* n_argv[] = {"ovms", "--config_path", "/unused", "--rest_port", "8080"};  // Workaround to have rest_port parsed in order to enable metrics
+        int arg_count = 7;
```

Comment:
Why 7?

Reply:
That's right, should be 5 (number of argument values)